content/docs/csidriver/installation/operator/redhat.md

```diff
@@ -0,0 +1,42 @@
+---
+title: CSI to CSM Operator Migration
+description: >
+  Migrating from CSI Operator to CSM Operator
+---
+
+## CR Sample Files
+
+{{<table "table table-striped table-bordered table-sm">}}
+|            |    CSI     |    CSM     |
+|------------|:----------:|:----------:|
+| PowerScale |    [isilon_v270_k8s_127.yaml](https://github.com/dell/dell-csi-operator/blob/main/samples/isilon_v270_k8s_127.yaml)    |     [storage_csm_powerscale_v280.yaml](https://github.com/dell/csm-operator/blob/main/samples/storage_csm_powerscale_v280.yaml)       |
+| PowerMax   |    [powermax_v270_k8s_127.yaml](https://github.com/dell/dell-csi-operator/blob/main/samples/powermax_v270_k8s_127.yaml)    |     [storage_csm_powermax_v280.yaml](https://github.com/dell/csm-operator/blob/main/samples/storage_csm_powermax_v280.yaml)       |
+| PowerStore |    [powerstore_v270_k8s_127.yaml](https://github.com/dell/dell-csi-operator/blob/main/samples/powerstore_v270_k8s_127.yaml)    |     [storage_csm_powerstore_v290.yaml](https://github.com/dell/csm-operator/blob/main/samples/storage_csm_powerstore_v290.yaml)       |
+| Unity XT      |    [unity_v270_k8s_127.yaml](https://github.com/dell/dell-csi-operator/blob/main/samples/unity_v270_k8s_127.yaml)    |     [storage_csm_unity_v280.yaml](https://github.com/dell/csm-operator/blob/main/samples/storage_csm_unity_v280.yaml)       |
+| PowerFlex  |     [vxflex_v270_k8s_127.yaml](https://github.com/dell/dell-csi-operator/blob/main/samples/vxflex_v270_k8s_127.yaml)       |      [storage_csm_powerflex_v290.yaml](https://github.com/dell/csm-operator/blob/main/samples/storage_csm_powerflex_v290.yaml)      |
+{{</table>}}
+
+## OpenShift Web Console Migration Steps
+
+1. Save the CR yaml file of the current CSI driver to preserve the settings (for use in step 6). Use the following commands in your cluster to get the CR:
+  ```bash
+      kubectl -n <namespace> get <CRD_kind>
+      kubectl -n <namespace> get <CRD_kind>/<CR_name> -o yaml
+  ```
```

Comment:
We should remove the `bash` because it puts a Copy button there but these commands can't be executed because of the placeholders.